h5py/_hl/files.py

```diff
@@ -39,6 +39,47 @@
 libver_dict_r = dict((y, x) for x, y in six.iteritems(libver_dict))
 
 
+def _set_fapl_mpio(plist, **kwargs):
+    kwds.setdefault('info', mpi4py.MPI.Info())
+    plist.set_fapl_mpio(**kwargs)
+
+
+drivers = {
+    'sec2': lambda plist, **kwargs: plist.set_fapl_sec2(**kwargs),
+    'stdio': lambda plist, **kwargs: plist.set_fapl_stdio(**kwargs),
+    'core': lambda plist, **kwargs: plist.set_fapl_core(**kwargs),
+    'family': lambda plist, **kwargs: plist.set_fapl_family(
+        memb_fapl=plist.copy(),
+        **kwargs,
```

Comment:
Extra comma (which is causing the tests to fail).

Reply:
ah, only tested on 3.6

Reply:
If you have multiple versions of python on your path, using tox helps catch that kinda thing (I know it's saved me from creating python 3 only fixes). Enabling travis and appveyor on your fork also helps (esp. appveyor if you lack easy access to windows).